pkg/azure/client/factory.go

```diff
@@ -19,6 +19,7 @@ import (
 
 	"github.com/gardener/gardener-extension-provider-azure/pkg/internal"
 
+	azurecompute "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-12-01/compute"
```

Comment:
Currently the provider imports `"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-12-01/compute"` and the infrastructure test imports `"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2020-06-01/compute"` which causes both versions of the package to be vendored. So we should stick to one of the versions and use it across the project.

After applying the following change on `test/integration/infrastructure/infrastructure_test.go`

```diff
-	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2020-06-01/compute"
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-12-01/compute"
```

```
$ make revendor

$ git diff --shortstat
 36 files changed, 1 insertion(+), 33388 deletions(-)
```

=> `"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2020-06-01/compute"` becomes obsolete

Reply:
I would do it the other way around though to get the newer version

Reply:
Yep, sure, makes sense.

Reply:
I will even go a step further and update the compute sdk to the most recent version `2020-06-30`.

Reply:
done